daml.yaml

```diff
@@ -4,7 +4,7 @@
 sdk-version: 2.5.0
 name: daml-finance
 source: src/test/daml
-version: 1.1.5
+version: 1.1.6
```

Comment:
When are you planning to push this change out? 

I would keep `1.1.x` for patches to the docs going out with `2.6` and use `1.2.x` otherwise, wdyt?

Reply:
I see i your other PR that you are targeting `2.7`, so I would go for `1.2.0`

Reply:
If we agree this is the way to go we should document it

Reply:
Sure, makes sense